### ISSUE TITLE
Keyboard tabbing order of tabset tabs and focusable controls within tabset content

### ISSUE THREAD

**Issue by jimjam-slam** [quarto-dev/quarto-cli#730]
I'm seeing some inconsistency with the keyboard tabbing order when I use tabsets.

In [this example](https://github.com/360-info/report-global-education/blob/main/embed-bycountry.qmd), in which I've styled the tabs to appear at the bottom of the page ([deployed version available here](https://360info-studentflows.pages.dev/embed-bycountry)), the tabs have `tabindex="0"` and so are tabbable - but tabbing to put focus on them activates them, so it's impossible to tab into focusable elements within the tabset content (eg. checkboxes) unless you click on one of them first.

In the following example, the tabs don't received any `tabindex` attribute at all and so aren't tabbable:

````
---
title: "Test tabs"
author: "jimjamslam"
---

Let's try some tabs!

::: {.panel-tabset}

## Tab A

This is just some explanatory text. No biggie...

## Tab B

Let's add some focusable controls here!

```{ojs}
viewof catSelect = Inputs.select(
  ["Apple", "Banana", "Orange", "Kiwi"],
  { label: "Choose a fruit"});

viewof numSelect = Inputs.range([1, 20], { label: "And a number" });
```

## Tab C

Some more text here! Could be lots of stuff.

```{ojs}
viewof check = Inputs.checkbox(
  ["Don't forget to check this!"],
  { label: "Options" })
```
## Tab D

Nah, this is all boring.

:::
````

Ideally, I expect that tabset tabs would be tabbable, but that tabbing onto a different tab would focus it without activating it so that I could continue on to any controls visible (unless I explicitly activated it as I might a hyperlink).

I'm not quite sure why the tabs in my first example have `tabindex` and the ones in my second don't, unless there's been a change in behaviour that I missed (although I believe I rendered them both with them same versions of everything).

I think I've said "tab" enough for the rest of the year now 😂

**Comment by jimjam-slam** [comment]
(I should note that [the tabs in the tabset documentation](https://quarto.org/docs/interactive/layout.html#tabset-panel) work as I expect, focusing on tab but not activating unless you press Enter! So maybe this is something I'm mucking up...)

**Comment by jimjam-slam** [comment]
Oh! One other difference with my first example is that it uses `theme: none`, which could be a problem if part but not all of the tabset functionality comes from Bootstrap...

**Comment by jjallaire** [comment]
Yeah `theme: none` ends up using a different tabbing library (https://github.com/cferdinandi/tabby) which have their own behavior (https://github.com/cferdinandi/tabby#keyboard-navigation)

**Comment by jimjam-slam** [comment]
> Yeah `theme: none` ends up using a different tabbing library (https://github.com/cferdinandi/tabby) which have their own behavior (https://github.com/cferdinandi/tabby#keyboard-navigation)

That makes sense! However, it's not what I'm seeing in my [first example](https://360info-studentflows.pages.dev/embed-bycountry), which should be using Tabby. There, when the first tab is focused, arrow keys do nothing (instead of activating the next tab) and the tab key activates the next tab (instead of moving focus to the tab content).
